src/geom/Transform.cc

```diff
@@ -105,16 +106,28 @@ typename FromEndpoint::Array Transform<FromEndpoint, ToEndpoint>::tranInverse(
 }
 
 template <typename FromEndpoint, typename ToEndpoint>
+Transform<ToEndpoint, FromEndpoint> Transform<FromEndpoint, ToEndpoint>::getInverse() const {
+    auto inverse = std::dynamic_pointer_cast<ast::FrameSet>(_frameSet->getInverse());
+    if (!inverse) {
+        // std::bad_cast more appropriate, but doesn't aid in debugging
```

Comment:
I like the code but find the note confusing. This situation should never occur, so logic_error does seem the best choice.